docs/Tutorials/Authentication/Methods/AzureAD.md

```diff
@@ -62,6 +62,38 @@ Start-PodeServer {
 }
 ```
 
+## Requests via Non-Modern Authentication Applications (for example PowerShell Invoke-RestMethod)
```

Comment:
Everything looks great, thanks for writing this up!

I'm just thinking the header might want a little shrinking - perhaps `Requests via CLI`, or `Requests using Basic Authentication`, and move the Invoke-RestMethod example to the description below? Open to ideas on the title 😄 

(saves the navigation to the right of the docs looking a little ugly, haha 😂)

Reply:
Thanks for the hint, didn't check the navigation as I only opened it up using VSCode's Markdown Preview.
I've used your second example and moved the actual example into the text.